cmd/algorand-indexer/api_config.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/algorand/indexer/api"
+	"github.com/algorand/indexer/api/generated/v2"
+	"github.com/algorand/indexer/config"
+)
+
+var apiConfigCmd = &cobra.Command{
+	Use:   "api-config",
+	Short: "dump api configuration",
+	Long:  "dump api configuration",
+	//Args:
+	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		config.BindFlags(cmd)
+		err = configureLogger()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to configure logger: %v", err)
+			os.Exit(1)
+		}
+		swag, err := generated.GetSwagger()
+
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to get swagger: %v", err)
+			os.Exit(1)
+		}
+
+		var displayDisabledMapConfig *api.DisplayDisabledMap
+		// Show a limited subset
+		if len(args) == 0 {
+			displayDisabledMapConfig = api.MakeDisplayDisabledMapFromConfig(swag, disabledMapConfig, true)
+
+		} else {
+			// This is the only acceptable option
+			if args[0] != "all" {
```

Comment:
We should adjust the options so that they are included in the documentation. Perhaps instead of "all" it is a bool `--all` flag. Is this going to be a problem with the other command that are planned?